source/scripts/Dude.js

```diff
@@ -27,15 +27,19 @@ export default class Dude extends Pixi.Sprite {
         
         if (Keyb.isDown("W")) {
             this.position.y -= this.speed.movement * delta.s
+            this.tint = 0xFF0000
```

Comment:
Multiple changes in a single CR. Please split.

Reply:
Good catch! Is it okay if I respect this in all CRs from now on?

Reply:
Next time there will be punishments...